### PR TITLE
Anchor menu item spec

### PR DESCRIPTION
**Pull request by m-akinc**
# Pull Request

## 🤨 Rationale

See #1020 

## 👩‍💻 Implementation

Refer to spec

## 🧪 Testing

N/A

## ✅ Checklist

- [x] I have updated the project documentation to reflect my changes or determined no changes are needed.

### CODE REVIEW

**Review by atmgrifter00 — APPROVED** [review]

**Review by mollykreis — APPROVED** [review]

File: packages/nimble-components/src/anchor-menu-item/specs/README.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+-   Supports icon
+-   Disabled state
+-   Follows menu item navigation pattern
+
+### Risks and Challenges
+
+None
+
+### Prior Art/Examples
```
Comment (jattasNI) [inline packages/nimble-components/src/anchor-menu-item/specs/README.md:27]: Do we have known use cases for this component in SLE or other clients? I think the answer is yes. Not a huge deal whether or not you document them here but at some point we should figure out what the integration story will be once this component is done.

Let me know if you want to track down use cases and capture them somewhere or if you'd like the Fred-Jesse PO conglomerate to take it on

File: packages/nimble-components/src/anchor-menu-item/specs/README.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+We will create a Blazor wrapper for the anchor menu item. There are no special routing considerations/mechanisms for Blazor.
+
+### Visual Appearance
+
+The component will look just like a standard menu item. See the design doc linked at the top of this file for details.
+
+---
+
+## Implementation
```
Comment (jattasNI) [inline packages/nimble-components/src/anchor-menu-item/specs/README.md:94]: Anything we need to think about / document about the expectations that the nimble-menu has about its items? It doesn't encode any direct dependencies on nimble-menu-item, does it?

**Review by jattasNI — APPROVED** [review]

File: packages/nimble-components/src/anchor-menu-item/specs/README.md (hunk 1 above)
Comment (mollykreis) [reply packages/nimble-components/src/anchor-menu-item/specs/README.md:27]: It's used in the help menu within SLE...or, at least it used to be. I don't even see the help menu on any apps right now.

File: packages/nimble-components/src/anchor-menu-item/specs/README.md (hunk 2 above)
Comment (m-akinc) [reply packages/nimble-components/src/anchor-menu-item/specs/README.md:94]: Not that I can find.

File: packages/nimble-components/src/anchor-menu-item/specs/README.md (hunk 1 above)
Comment (m-akinc) [reply packages/nimble-components/src/anchor-menu-item/specs/README.md:27]: I don't immediately know how to find instances of it, but I'll give it a try and ask for help if I need it.

**Pull request merged**